lib/static/components/suites.jsx

```diff
@@ -112,7 +112,16 @@ function Suites(props) {
 
     const {visibleRootSuiteIds} = props;
 
-    if (isEmpty(visibleRootSuiteIds)) {
+    if (props.isReportEmpty) {
+        return <div className="sections sections_type_empty">
```

Comment:
I would pull this `html` (as well as "There are no tests that match to selected filters" html) out to functions, so here it would look like:

```js
if (props.isReportEmpty) {
    return renderEmptyReport();
}

if (isEmpty(visibleRootSuiteIds)) {
  return renderEmptyVisibleRootSuiteIds(_getSelectedFiltersInfo());
}

return (
  <div className="sections">
  ...
)

Reply:
IMO it only sacrifices readability without solving any actual problems. Keeping it as is for now.